src/components/projects/create-wp-form/create-wp-form.tsx

```diff
@@ -75,20 +98,88 @@ const CreateWPForm: React.FC = () => {
     }
   };
 
-  const handleSubmit = (e: any) => {
+  const stateBundleBuilder = (state: any, setter: any) => {
+    return {
+      state,
+      setter
+    };
+  }
+
+  const states = {
+    name: stateBundleBuilder(name, setName),
+    wbsNum: stateBundleBuilder(projectWbsNum, setWbsNum),
+    crId: stateBundleBuilder(crId, setCrId),
+    startDate: stateBundleBuilder(startDate, setStartDate),
+    duration: stateBundleBuilder(duration, setDuration)
+  };
+
+  const handleSubmit = async (e: any) => {
     e.preventDefault();
 
     // exits handleSubmit if form input invalid (should be changed in wire up)
-    const wbsNum: WbsNumber = validateWBS(e.target.wbsNum.value.trim());
+    let wbsNum: WbsNumber;
+    try {
+      wbsNum = validateWBS(projectWbsNum);
 
-    if (!isProject(wbsNum!)) {
-      alert('Please enter a valid Project WBS Number.');
-      return;
+      const { userId } = authContext!.user!;
+
+      if (!isProject(wbsNum!)) {
+        alert('Please enter a valid Project WBS Number.');
+        return;
+      }
+      const depWbsNums = dependencies.map(dependency => {
+        const depWbsNum = validateWBS(dependency.trim());
+        return {
+          carNumber: depWbsNum.car,
+          projectNumber: depWbsNum.project,
+          workPackageNumber: depWbsNum.workPackage
+        }
+      });
+      if (depWbsNums) {
```

Comment:
The typings say that `depWbsNums` will always result in an array, so it will always be truthy and the else won't be reached, not sure what you're testing here